### DISCUSSION TITLE
Integration with zabbix

### DISCUSSION THREAD

**Discussion by pramod-desai-minutus** [apache/skywalking#10318] [Q&A] [answered]
Hi there,

I am new to skywalking, How to do configuration to get zabbix metrics in skywalking.
Any documentaion or mentioning steps would be better

Thanks in advance!!

**Comment by Superskyyy** [comment, marked as answer]
https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-zabbix/ @Pramod6919 Hello please check this one out.

**Reply by pramod-desai-minutus** [reply]
@Superskyyy  Thank you, Worked!
